### PR TITLE
NETOBSERV-1091: remove CO-RE file and extensions as that causes douple allocations

### PR DESCRIPTION
**Pull request by msherif1234**
it was noticed cpu and memory are higher than 1.2 release 
that change was triggered by #110 
it seems using co-re results in a significant memory and cpu increase https://github.com/libbpf/libbpf/issues/697 is opened against libbpf to track this, issue has been seen even with latest version of the lib, we don't use CO-RE apis at all so for now we will remove it and once we have libbpf fix we will re-enable

diag on left side with this PR changes the diag on right is with 1.3 baseline
image
image

### CODE REVIEW

**Comment by msherif1234** [comment]
https://github.com/libbpf/libbpf/issues/697

File: bpf/flows.c [hunk 1]
```diff
@@ -13,11 +13,11 @@
             until an entry is available.
         4) When hash collision is detected, we send the new entry to userpace via ringbuffer.
 */
+#define BPF_NO_PRESERVE_ACCESS_INDEX
```
Comment (jotak) [inline bpf/flows.c:16]: can you add a comment explaining briefly the reason (or just linking to the JIRA and/or libpbf issue)  ?

**Comment by jotak** [comment]
I'm fine to merge it and also backport on 1.3
@msherif1234 can you open the backport PR as well ?

**Comment by msherif1234** [comment]
> I'm fine to merge it and also backport on 1.3 @msherif1234 can you open the backport PR as well ?

I linked the issue as well as the PR that trigger this in the PR description shouldn't that be enough ?

**Comment by jotak** [comment]
/approve
muchas gracias @msherif1234 !!!

**Pull request merged**